examples/server/server.cpp

```diff
@@ -2390,12 +2391,13 @@ static void server_params_parse(int argc, char **argv, server_params &sparams,
                 break;
             }
             std::string value(argv[i]);
```

Comment:
`value` seems unused now?

Reply:
Yeah it's unused, I forgot to remove that. It's now removed